auth/src/main.ts

```diff
@@ -0,0 +1,10 @@
+import { ValidationPipe } from '@nestjs/common';
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
```

Comment:
swagger or nah?

Reply:
https://github.com/orgs/world-wide-weights/projects/1?pane=issue&itemId=17571943

Will be added here